SupportSDKSamples/CocoaPodSample/AppDelegate.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  AppDelegate.swift
+//  CocoaPodSample
+//
+//  Created by Killian Smith on 22/05/2018.
+//  Copyright © 2018 Killian Smith. All rights reserved.
+//
+
+import UIKit
+import ZendeskSDK
+import ZendeskCoreSDK
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    
+    var window: UIWindow?
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+        CoreLogger.enabled = true
+        CoreLogger.logLevel = .verbose
+        
+        Zendesk.initialize(appId: "appId",
```

Comment:
@KillianSmith0 Could you enclose the config params in Xcode recognisable tags <#param#>? This way everybody who will try to use the sample will be prompted by the compiler to provide real data.